src/aarch64_aol.properties

```diff
@@ -36,3 +36,37 @@ aarch64.MacOSX.gpp.shared.extension=dylib
 aarch64.MacOSX.gpp.plugin.extension=bundle
 aarch64.MacOSX.gpp.jni.extension=jnilib
 aarch64.MacOSX.gpp.executable.extension=
+
+
+#
+# aarch64 (arm64) Linux
+#
+aarch64.Linux.linker=g++
+
+aarch64.Linux.gpp.c.compiler=gcc
+aarch64.Linux.gpp.c.defines=Linux GNU_GCC
+aarch64.Linux.gpp.c.options=-Wall -Wno-long-long -Wpointer-arith -Wconversion -fPIC
+aarch64.Linux.gpp.c.includes=**/*.c
+aarch64.Linux.gpp.c.excludes=
+
+# options for gcc linker front end
+aarch64.Linux.gpp.linker.options=-shared -shared-libgcc -fPIC -fexceptions
+aarch64.Linux.gpp.linker.sysLibs=stdc++
+aarch64.Linux.gpp.linker.libs=
+
+aarch64.Linux.gpp.java.include=include;include/linux
+aarch64.Linux.gpp.java.runtimeDirectory=jre/lib/i386/client
+
+aarch64.Linux.gpp.lib.prefix=lib
+aarch64.Linux.gpp.shared.prefix=lib
+aarch64.Linux.gpp.static.extension=a
+aarch64.Linux.gpp.shared.extension=so
+aarch64.Linux.gpp.plugin.extension=so
+aarch64.Linux.gpp.jni.extension=so
+aarch64.Linux.gpp.executable.extension=
+
+# FIXME to be removed when NAR-6
+aarch64.Linux.gcc.static.extension=a
+aarch64.Linux.gcc.shared.extension=so*
+aarch64.Linux.gcc.plugin.extension=so
+aarch64.Linux.gcc.jni.extension=so
```

Comment:
We could also add the properties to build for Mac M1: https://github.com/maven-nar/nar-maven-plugin/pull/382/files#diff-f382ee447abafda33869af1a7255d02f28b8b0cb55b9a0c91bd30cabbfbdce82R1349

Reply:
@merlimat We have supported M1: https://github.com/apache/bookkeeper/pull/3175/files